### DISCUSSION TITLE
GameSDK compatibility?

### DISCUSSION THREAD

**Discussion by VinnyVicious** [o3de/o3de#1886] [Q&A]
The basic sample project from Lumberyard, GameSDK, is apparently incompatible with O3DE. Is there an alternative, or documentation to follow on how to port Lumberyard projects to O3DE?

https://docs.aws.amazon.com/lumberyard/latest/userguide/sample-project-legacy.html

**Comment by Infurael** [comment]
The following projects are currently available for testing and examples:
- Atom Sample Viewer: https://github.com/o3de/o3de-atom-sampleviewer
- Atom Test: https://github.com/o3de/o3de-atomtest
- Network Soak Test: https://github.com/o3de/o3de-netsoaktest
- Multiplayer Sample: https://github.com/o3de/o3de-multiplayersample
Many of the projects from LY do not work if they use deprecated features such as the CryRenderer. @amzn-melissaa and her team are working on converting some of the LY samples over to O3DE. Stay tuned for more tutorials, videos and samples in the future.
